addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss

```diff
@@ -20,3 +20,14 @@
         opacity: 1;
     }
 }
+// TODO: Put the following rules in a file that is not sent to the visitor.
+body.editor_enable {
+    .s_newsletter_subscribe_form {
+        .o_enable_preview {
+            display: block !important;
+        }
+        .o_disable_preview {
+            display: none !important;
+        }
+    }
+}
```

Comment:
Detail, but since you will need to modify this PR a bit: as a stable fix, I would prefer you put these rules in a website editor file (even though it's a website_mass_mailing rule). That way you don't impact stable version visitors. Totally a detail in this case, but just so you know.

Note: in 15.0, you would be able to add a new file, so there won't ever be that kind of problem anymore anyway.